src/GregClient/packages.config

```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="8.0.2" targetFramework="net40" />
+  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net48" />
```

Comment:
target fx is net48 here but above, in the csproj, the path is pointing to net45?
`..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll`

Reply:
I can change that, on this case the packages.config is the one used on the solution restore build task

Reply:
Yes, the target on the packages.config file is because I've change the target framework on the project to 4.8, and I've got some warnings on the building related to the mismatch

On the csproj, there´s only the reference of the path where you can find the nearest version to the target framework of that nuget package assembly, for example, NewtonSoft.Json 8.0.3 was targeted from 2, to 4.5, that is the nearest to our target framework and that's the reason you see that one on the HintPath
![image](https://user-images.githubusercontent.com/47230272/86671403-39ef6000-bfbb-11ea-8204-1f4c5520fe91.png)